src/excmds.ts

```diff
@@ -2171,6 +2171,132 @@ export function unset(...keys: string[]) {
     config.unset(...target)
 }
 
+/** Include and toggle css style to a web page.
+ *
+ * Syntax: style {mode} [...args]
+ *
+ * @param mode
+ *         ALIASES          MODES
+ *      styleinclude   : style -i
+ *      styledelete    : style -x
+ *      styledisable   : style -d
+ *      styleenable    : style -e
+ *      styletoggle    : style -t
+ *
+ *  - `-i` includes new rule
```

Comment:
@caputchinefrobles: Could you try to explain this a little better? What does "include" mean? Where is it included?

Does it just mean "create"?